docs/modules/ROOT/pages/tutorial-standalone.adoc

```diff
@@ -0,0 +1,371 @@
+
+In this tutorial, we will use Docker to configure all the components individually from scratch.
+However, if you already have some of these components deployed, you can skip the respective sections. 
+You may wish to read xref:integrating-with-existing-deployments.adoc[].
+
+== Pre-Requisites
+
+We assume that you already have a Couchbase Server cluster running.
+If not, refer to the xref:7.0@server:install:getting-started[installation options] to deploy a cluster.
+
+We also assume that you have Docker installed.
+
+Create a named Docker network to enable the components to resolve each other by their hostnames:
+
+[source, console]
+----
+docker network create cmos
+----
+
+== Install Cluster Monitor
+
+TODO (https://issues.couchbase.com/browse/CMOS-313)
+
+== Install Prometheus
+
+First, create a configuration file for Prometheus - by convention this is called `prometheus.yml`.
+It will need to reference all your Couchbase Server nodes.
+
+.prometheus.yml
+[source, yaml]
+----
+global:
+  scrape_interval: 30s
+
+scrape_configs:
+  - job_name: couchbase-server
+    basic_auth:
+      username: Administrator <.>
+      password: password <.>
+    static_configs:
+      - targets:
+          - 10.145.212.101:8091 <.>
+          - 10.145.212.102:8091
+          - 10.145.212.103:8091
+        labels:
+          cluster: "Couchbase Server Cluster" <.>
+----
+
+<.> This needs to be set to the username of a Couchbase Server user with at least "External Stats Reader" permission.
+<.> This needs to be set to the password of the above user.
+<.> Here we specify the addresses of all the nodes in your cluster.
+    The port number needs to be the management port - use 18091 if you have TLS enabled.
+    There are other ways of configuring it that do not require hard-coding the addresses - refer to the https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config[Prometheus documentation^] for more details.
+<.> This label will be added to all metrics from this cluster.
+    The value should match the name the cluster is configured to use in the Web Console.
+    If you wish, you can add additional labels here (e.g., environment, datacenter...)
+
+Then, run Prometheus, mounting your configuration file inside the container.
+If you want to persist your data across restarts, you can also mount the default data directory.
+
+[source, console]
+----
+docker run -d \
+  -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+  -v /tmp/prometheus/data:/prometheus \ # Remove this if you do not want to persist the data.
+  -p 9090:9090 \
+  --network cmos \
+  --name prometheus \
+  prom/prometheus
+----
+
+If it has started successfully, you should be able to access its UI and see your cluster listed on http://localhost:9090/targets.
+
+
+=== Prometheus Alerting Rules
+
+You can reuse the CMOS alert definitions to get alerts about events in your cluster.
```

Comment:
NIT:
```suggestion
You can reuse inbuilt/ready made CMOS alert definitions to get alerts about events in your cluster.
```
wasn't there a discussion about whether customers should reuse our alert definitions??

Reply:
I think it's mostly a starting point as an example of the potential, which can be customised to suit etc.